pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -2046,7 +2046,7 @@ class Build {
                                 if (isPodman == 0) {
                                     // Note: --userns was introduced in podman 4.3.0
                                     // Add uid and gid userns mapping required for podman
-                                    dockerRunArg += " --userns keep-id:uid=1000,gid=1000"
+                                    dockerRunArg += " --userns keep-id:uid=1002,gid=1003"
```

Comment:
Has `--userns=keep-id` been considered? When running `podman` with the `id:gid` of the desired user (i.e. `1002:1003` in this case), explicit `uid`, `gid` mappings wouldn't be needed.

Reply:
Simpler answer: No. This was intended as a tactical fix based on what Andrew had done previously to get something live until I had the time to look at other options, of which your suggestion sounds like a good one to look at.

Reply:
OK.